accelerator/core/background.c

```diff
@@ -0,0 +1,260 @@
+#include "background.h"
+
+#define BK_LOGGER "core"
+
+static logger_id_t logger_id;
+
+void bk_logger_init() { logger_id = logger_helper_enable(BK_LOGGER, LOGGER_DEBUG, true); }
+
+int bk_logger_release() {
+  logger_helper_release(logger_id);
+  if (logger_helper_destroy() != RC_OK) {
+    ta_log_error("Destroying logger failed %s.\n", BK_LOGGER);
+    return EXIT_FAILURE;
+  }
+
+  return 0;
```

Comment:
Instead of returning 0, use enumeration.

Reply:
I will open another issue for this fix, since all the logger releasing functions in tangle-accelerator return `int` instead of enumeration